oat-sa/lib-lti1p3-core
# Feature/tr 4408/redirect uri must be static

# [TR-4408](https://oat-sa.atlassian.net/browse/TR-4408)

## How to test
1. Create delivery
2. Generate delivery URL 
3. Check authentication response parameters
4. We Should have in redirect_uri static url

## Development impact 
1. Modified `OidcInitiator` for detect dynamic url end 
3. Added new constructor param $isRemoveDynamicRedirectUriPart 
4. By default false
5. If set to true it will remove dynamic part from LtiMessage for redirect_uri param 

### Demo video 
https://www.loom.com/share/0f8c9994de3c4da783273e300b0958f1
